packages/cli/src/commands/new.ts

```diff
@@ -37,6 +38,11 @@ export default class New extends Command {
       default: hasbin.sync('yarn'),
       allowNo: true,
     }),
+    "skip-install": flags.boolean({
```

Comment:
can we find a way to make this option undocumented? i'm not sure we want to expose this to users if it's just for testing. maybe do this as an environment variable instead? `BLITZ_SKIP_INSTALL=true blitz new my-proj`?

Reply:
We could this undocumented by setting `hidden: false` for the flag, which would simply hide it from `blitz new --help`.
I actually wondered wether this is helpful for testing purposes only - isn't there some use case where a user would want to skip installations? I cannot come up with one on the spot, but still ... 😅

Reply:
@aem so do we want to simply hide it or remove the option altogether and use an environment variable instead?

Reply:
Oops I thought I had commented back, sorry about that!! We can leave it like this but let’s still hide the arg from the docs. Thanks!